<?xml version="1.0"?>
<launch>
	<node pkg="service_demo" type="server_demo.py" output="screen" name="server_demo" required="true" />

	<node pkg="service_demo" type="client_demo.py" output="screen" name="client_demo" required="true" />

</launch>